tools/spelling/spelling_dictionary.txt

```diff
@@ -756,6 +756,7 @@ interpretable
 intra
 ints
 invariance
+iov
```

Comment:
This is probably not needed.

Reply:
Indeed the check fails. This is weird as I used iov in my PRs and everything was ok. And it's all over the code base. Looks like the code is treated as a comment by the spellchecker. @phlax could you please have a look?